boards/silabs/radio_boards/slwrb4180b/slwrb4180b.dts

```diff
@@ -0,0 +1,146 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Capgemini
+ * Author: Arunmani Alagarsamy <arunmani.a@capgemini.com>
```

Comment:
Generally, headers only mention Copyright holders. The author can be found using `git blame`.